app/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,7 @@
     <string name="main_activity_settings_password">Password</string>
     <string name="main_activity_settings_start_on_boot">Start on Boot</string>
     <string name="main_activity_settings_scaling">Scaling</string>
+    <string name="main_activity_settings_remove_padding">Remove padding</string>
```

Comment:
please make it "Remove Padding" (in title case). As an extra bonus, you could add this to the other translation as well using Google Translate :heart: .

Reply:
Added in ba78bb6